src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/naming/illegalidentifiername/InputIllegalIdentifierNameWhen.java

```diff
@@ -0,0 +1,54 @@
+/*
+IllegalIdentifierName
+format = (default)(?i)^(?!(record|yield|var|permits|sealed|when|_)$).+$
+tokens = (default)CLASS_DEF, INTERFACE_DEF, ENUM_DEF, ANNOTATION_DEF, ANNOTATION_FIELD_DEF, \
+         PARAMETER_DEF, VARIABLE_DEF, METHOD_DEF, ENUM_CONSTANT_DEF, PATTERN_VARIABLE_DEF, \
+         RECORD_DEF, RECORD_COMPONENT_DEF, LAMBDA
+
+
+*/
+
+//non-compiled with javac: Compilable with Java19
+package com.puppycrawl.tools.checkstyle.checks.naming.illegalidentifiername;
+
+public class InputIllegalIdentifierNameWhen {
+
+    int When = 10; // violation
+    int when = 10; // violation
+
+    int otherwhen = 11; // ok
+}
+
+class InputJava19PatternsTrickyWhenUsage {
+    static String m1(Object when) { // violation
```

Comment:
add message to all "// violation"

Do you any suggestion on how to enforce this rule by Checkstyle and put in suppression all existing files, to let us violate new files.

Reply:
it is becoming difficult to put violation message here bcs msg is too long if we use violation above and below still message become longer than 100 line, 
although if we not consider the 100 line it is failing only with 
```
Actual and expected violations differ.
expected to match: 16:(?:\d+:)?\s.*Name 'When' must match pattern '\(?i\)^\(?!\(record|yield|var|permits|sealed|when|_\)$\).+$'.*
but was          : 16:9: Name 'When' must match pattern '(?i)^(?!(record|yield|var|permits|sealed|when|_)$).+$'.
```  

we can write the message here something like this 
`    int When = 10; // violation 'Name 'When' must match pattern .*' ` this is working fine is it good to show message like this 

> Do you any suggestion on how to enforce this rule by Checkstyle and put in suppression all existing files, to let us violate new files.

i have to think on this right know i have not any suggestion once the issue https://github.com/checkstyle/checkstyle/issues/11214 will resolve and all the suppression from bdd is removed then if we not write the violation message on that file the mvn it would not pass 
right know  i want have any idea

Reply:
AS of know message written